Source/ThirdParty/PlayCppSdkLibrary/PlayCppSdkLibrary.Build.cs

```diff
@@ -39,12 +39,19 @@ public class PlayCppSdkLibrary : ModuleRules {
       PublicAdditionalLibraries.Add("crypt32.lib");
       PublicAdditionalLibraries.Add("ws2_32.lib");
 
-    } else if (UnrealTargetPlatform.Mac == Target.Platform) {
+    } else if (Target.Platform == UnrealTargetPlatform.Mac) {
       PublicAdditionalLibraries.Add(
           Path.Combine(LibrariesPath, "Mac", "libplay_cpp_sdk.a"));
-    } else if (UnrealTargetPlatform.Linux == Target.Platform) {
+    } else if (Target.Platform == UnrealTargetPlatform.Linux) {
       PublicAdditionalLibraries.Add(
           Path.Combine(LibrariesPath, "Linux", "libplay_cpp_sdk.a"));
+    } else if (Target.Platform == UnrealTargetPlatform.Android) {
+      PublicAdditionalLibraries.Add(
+          Path.Combine(LibrariesPath, "Android", "arm64-v8a", "libplay_cpp_sdk.a"));
+      PublicAdditionalLibraries.Add(
+          Path.Combine(LibrariesPath, "Android", "armeabi-v7a", "libplay_cpp_sdk.a"));
+      PublicAdditionalLibraries.Add(
+          Path.Combine(LibrariesPath, "Android", "x86_64", "libplay_cpp_sdk.a"));
     }
```

Comment:
should this also check the target architecture?

Reply:
The architecture is checked by the corresponding folder name, arm64-v8a, armeabi-v7a, x86_64. I can not find the official document how to check architecture, but other projects all are checking in this way. See https://github.com/search?p=2&q=UnrealTargetPlatform+Android+arm64-v8a&type=Code

Reply:
https://docs.unrealengine.com/4.27/en-US/ProductionPipelines/BuildTools/UnrealBuildTool/TargetFiles/ 

`Target.Architecture` should be the property to check, but it's a string

Reply:
but yeah, it seems it's common to bundle all architectures for android in a build script